### PR TITLE
Fix deployment: python + GAMS dependencies

### PR DESCRIPTION
**Pull request by schweikart**

### CODE REVIEW

File: qiskit/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# This file describes the python package requirements for all Qiskit scripts
+# suppored by ProvideQ
```
Comment (Elscrux) [inline qiskit/requirements.txt:2]:
```suggestion
# supported by ProvideQ
```

File: gams/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# This file describes the python package requirements for all GAMS scripts
+# suppored by ProvideQ
```
Comment (Elscrux) [inline gams/requirements.txt:2]:
```suggestion
# supported by ProvideQ
```

File: .gitignore [hunk 3]
```diff
@@ -40,3 +40,4 @@ out/
 ### Toolbox ###
 /jobs/
 gamslice.txt
+*.lst
```
Comment (Elscrux) [inline .gitignore:43]: Maybe separate a GAMS section with comment here. `.lst` could be anything

**Review by Elscrux — APPROVED** [review]
Runs for me. I just spotted a few minor things that could be changed.

File: qiskit/requirements.txt (hunk 1 above)
Comment (schweikart) [reply qiskit/requirements.txt]: fixed typo

File: gams/requirements.txt (hunk 2 above)
Comment (schweikart) [reply gams/requirements.txt]: fixed typo

**Pull request merged**